mautic/mautic
# Add created and modified data to contact profile

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Just add Created on  and Last modified to contact profile



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Check contact profile - core tab - new data  (Created on, Modified by, Last modified)

![image](https://user-images.githubusercontent.com/462477/180648517-d14d9e52-af9c-4518-962b-f6bdf9f39886.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11352"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
